aarch64/src/l.S

```diff
@@ -0,0 +1,29 @@
+// Aarch64 entry (Raspberry Pi 3, 4 focussed)
+ 
+.section .boottext, "awx"
+.globl start
+
+start:
+	// All cores other than 0 should just hang
+    mrs     x1, mpidr_el1
+    and     x1, x1, #3
```

Comment:
If we have a board with more than 4 HW threads, this'll break, no? This doesn't seem to match the description in the ARM datasheet for mpidr_el1:

https://developer.arm.com/documentation/100403/0301/register-descriptions/aarch64-system-registers/mpidr-el1--multiprocessor-affinity-register--el1